src/scenic/core/distributions.py

```diff
@@ -796,23 +801,27 @@ def scalar(thing):
     def sampleGiven(self, value):
         first = value[self.object]
         rest = [value[child] for child in self.operands]
+        kwargs = {key: value[child] for key, child in self.kwoperands.items()}
         op = getattr(first, self.operator)
-        result = op(*rest)
+        result = op(*rest, **kwargs)
         if result is NotImplemented and self.reverse:
-            assert len(rest) == 1
+            assert len(rest) == 1 and len(kwargs) == 0
             rop = getattr(rest[0], self.reverse)
             result = rop(first)
         if result is NotImplemented and self.symbol:
             raise TypeError(
                 f"unsupported operand type(s) for {self.symbol}: "
-                f"'{type(first).__name__}' and '{type(rest[0]).__name__}'"
+                f"'{type(first).__name__}', '{type(rest[0]).__name__}'"
```

Comment:
Why was this changed? This error message comes from Python (try `4+"a"`).

Reply:
Huh, I have no idea. I'll change it back.